src/tools/status_tool.cpp

```diff
@@ -780,12 +766,121 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
         }
     }
     std::cout << "])" << std::endl;
+    return ret;
+}
+
+int GetChunkserverLeftSize(
+    const std::map<PoolIdType, std::vector<ChunkServerInfo>> &poolChunkservers,
+    std::map<PoolIdType, std::vector<uint64_t>> *poolChunkLeftSize,
+    std::map<PoolIdType, std::vector<uint64_t>> *poolWalSegmentLeftSize,
+    bool *useWalPool,
+    bool *useChunkFilePoolAsWalPool,
```

Comment:
'useWalPool' and 'useChunkFilePoolAsWalPool' may not pass by pointer，it can pass by value if we do not change it in this function.

Reply:
> 'useWalPool' and 'useChunkFilePoolAsWalPool' may not pass by pointer，it can pass by value if we do not change it in this function.

done